source/connect/favorite-connections/import-export-ui/export.txt

```diff
@@ -0,0 +1,113 @@
+.. _compass-export-connections-ui:
+
+==============================================
+Export Favorite Connections in |compass-short|
+==============================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+You can export favorite connections in |compass|. Colleagues can import your 
+favorite connections, or you can use them in your other workspaces.
+
+About This Task
+---------------
+
+.. include:: /includes/fact-export-favorite-connection.rst
+
+Procedure
+~~~~~~~~~
```

Comment:
I think we'll need to update all of the headings on this page to use dashes `-`. Currently, all of the remaining sections appear as sub-heading of "About This Task".

Reply:
Updated